src/wiki/plugins/images/apps.py

```diff
@@ -1,8 +1,15 @@
-from django.apps import AppConfig
+from django.apps import AppConfig, apps
+from django.core.exceptions import ImproperlyConfigured
 from django.utils.translation import gettext_lazy as _
 
 
 class ImagesConfig(AppConfig):
     name = 'wiki.plugins.images'
     verbose_name = _("Wiki images")
     label = 'wiki_images'
+
+    def ready(self):
+        if not apps.is_installed("sorl.thumbnail"):
```

Comment:
This will not use the django check framework.
Better have a checks module which export a function and then register with check framework.
Something as:
```python
rom django.apps import apps
from django.core.checks import Error
class Tags:
  required_installed_apps = "required_installed_apps"
def check_for_required_installed_apps(app_configs, **kwargs):
  errors = []
  if not apps.is_insalled("sorl.thumbnail"):
    errors.append(Error('needs sorl.thumbnail in INSTALLED_APPS', id='wiki_images.E001'))
  return errors
```
and then register them in `ready()`
```python
from django.core.checks import register
from .import checks
def ready(self):
  register(checks.check_for_required_installed_apps, checks.Tags.required_installed_apps)
```

Reply:
Thank you.